### PR TITLE
Attempted to fix all the deployment errors

### PR DESCRIPTION
**Pull request by christos-c**
Still not sure if everything is working properly.

### CODE REVIEW

File: lbjava/src/test/java/edu/illinois/cs/cogcomp/lbjava/MainTest.java [hunk 1]
```diff
@@ -165,7 +165,7 @@ public void cleanup() {
         File[] dirFiles = lbjDir.listFiles(new FilenameFilter() {
 
             public boolean accept(File dir, String name) {
-                return !name.endsWith(".lbj");
+                return !name.endsWith(".lbj") && !name.startsWith(".nfs");
```
Comment (danyaljj) [inline lbjava/src/test/java/edu/illinois/cs/cogcomp/lbjava/MainTest.java]: What is the reason for this change?

Comment (christos-c) [inline lbjava/src/test/java/edu/illinois/cs/cogcomp/lbjava/MainTest.java]: My desktop couldn't delete those files (I'm guessing temp filesystem files)
